thirdparty_containers/HeapFragger/playlist.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!--
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#      https://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+-->
+<playlist xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:noNamespaceSchemaLocation="../TestConfig/playlist.xsd">
+	<test>
+		<testCaseName>heapfragger</testCaseName>
+	<command>docker run --rm $(EXTRA_DOCKER_ARGS) adoptopenjdk-heapfragger:latest ; \
+		$(TEST_STATUS); \
+		docker rmi -f adoptopenjdk-heapfragger
+	</command>
+		<subsets>
+			<subset>8</subset>
+		</subsets>
+		<levels>
+			<level>sanity</level>
+		</levels>
+		<groups>
+			<group>external</group>
+		</groups>
```

Comment:
If this is for openj9 implementation, name this one heapfragger_j9 and add `<impls><impl>openj9</impl></impls>` .

For hotspot support, we should add a 2nd test target to the playlist called heapfragger_hs, and can use the approach of passing args in via `<command>` tag.  

See system-test as an example:
https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/thirdparty_containers/system-test/playlist.xml#L18
https://github.com/AdoptOpenJDK/openjdk-tests/blob/master/thirdparty_containers/system-test/dockerfile/system-test.sh#L59

Reply:
Impl and test name changed. I'll look into getting hotspot and adding it as a 2nd test target